index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header class="container">
+        <div class="logo">My Logo</div>
```

Comment:
You did a good job in terms of the header layout design, and it's almost very similar to the original Figma design. But you just need to remove the extra space from the layout to make it look the same.

![image](https://user-images.githubusercontent.com/34779928/191851330-f4bc573f-194e-4706-a9b5-8a9a715a79bf.png)